CHANGELOG

```diff
@@ -1,6 +1,18 @@
 devel
 -----
 
+* Updated internal JavaScript dependencies:
+
+  - @xmldom/xmldom: 8.6.0 -> 0.8.6
```

Comment:
It looks like we are going from version number 8.6 down to 0.8.6.
Pretty sure there is a release for this, but it looks weird.